test/e2e/framework/volume/fixtures.go

```diff
@@ -89,6 +89,12 @@ const (
 	iSCSIIQNTemplate = "iqn.2003-01.io.k8s:e2e.%s"
 )
 
+// SizeRange encapsulates a range of sizes specified as minimum and maximum quantity strings
+type SizeRange struct {
+	Max string // Max quantity specified as a string including units. E.g "3Gi"
+	Min string // Min quantity specified as a string including units. E.g "1Gi"
+}
```

Comment:
Can you add a comment saying that both values are optional and that which global default is used if unset?

Reply:
+1 with @pohly